server/controllers/stripecheckout.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const router = express.Router();
+require("dotenv").config();
+
+//secret api key in .env
+const stripe = require('stripe')(process.env.SECRET_KEY);
+
+//test for route
+router.get("/test", async(req, res) => {
+  res.send("working");
+});
+
+//stripe checkout session endpoint
+router.post("/create-checkout-session", async (req, res) => {
```

Comment:
"POST" means "create" so double meaning.
Please change endpoint name.
ex) "/checkoutsession"

Reply:
Thank you, I updated the endpoint to `/checkoutsession` 👍